rdr_service/genomic/genomic_job_components.py

```diff
@@ -2543,6 +2638,32 @@ def _get_new_c1_participants(self, from_date):
 
         return list([list(r) for r in zip(*result)])
 
+    def _get_long_read_participants(self):
+        """
+        Retrieves participants based on filters that have
+        been denoted to use in the long read pilot program
+        """
+        with self.member_dao.session() as session:
+            result = session.query(GenomicSetMember).join(
+                ParticipantSummary,
+                GenomicSetMember.participantId == ParticipantSummary.participantId,
+            ).join(
+                ParticipantRaceAnswers,
+                ParticipantRaceAnswers.participantId == ParticipantSummary.participantId,
+            ).join(
+                Code,
+                ParticipantRaceAnswers.codeId == Code.codeId,
+            ).filter(
+                Code.value == 'WhatRaceEthnicity_Black',
+                GenomicSetMember.genomeType == 'aou_wgs',
+                GenomicSetMember.genomeType != 'long_read',
+                GenomicSetMember.genomicWorkflowState != GenomicWorkflowState.IGNORE,
+                ParticipantSummary.participantOrigin == 'vibrent',
+                ParticipantSummary.ehrUpdateTime.isnot(None)
```

Comment:
I think Sunny's idea was to use participants that had recent EHR update times. From what I understand the hope is to get participants that have submitted alot of EHR files, but we just don't know the number of distinct files. We could use the latest, but that could get really new participants that have just submitted one.

What if we ordered the list of participants by the amount of time between their first EHR upload (`ehrReceiptTime`) and their latest upload (`ehrUpdateTime`) with the biggest time spans at the top?

Reply:
Yea discussed it with Josh, just going with checking for `ehrUpdateTime` at this time.